packages/utils/src/helpers.ts

```diff
@@ -11,20 +11,19 @@
 //    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
-//@ts-nocheck
 import dayjs from 'dayjs';
 import utc from 'dayjs/plugin/utc.js';
 import pluralize from 'pluralize';
 import Cookies from 'universal-cookie';
 
 dayjs.extend(utc);
 
-const isEncoded = uri => {
+const isEncoded = (uri: string) => {
   uri = uri || '';
```

Comment:
Not related to your changes, but this could be achieved through default params. I think it's better to avoid reassigning the arguments

Reply:
we can't default to this everywhere though, as we might get a falsy value that's not `undefined` passed (thinking primarily about the inputs where we fallback to empty strings)